src/fontra/client/core/experimental-features.js

```diff
@@ -0,0 +1,11 @@
+import { ObservableController } from "./observable-object.js";
+
+export const experimentalFeaturesController = new ObservableController({
+  scalingEditBehavior: false,
```

Comment:
`scalingEditBehavior` can be a property on both tools objects, its value being `false` on `PointerTool`, and `true` on `PointerToolScale`.

Reply:
I assume the same can be done for `quadPenTool` (maybe renamed as `drawQuadratic`) for the two pen tools.

Reply:
I changed the way 'scalingEditBehavior' works and removed scalingEditBehavior and quadPenTool from controller and gear panel. I did not rename quadPenTool, because it's redundant anyway as it's a new separate tool. 

Even though it feels like I'm done with it, I keep this feedback open because I'm not 100% sure I've managed to change it the way you want it to be.

Reply:
I assume it's save to close this as we have solved it with [ee0f88c](https://github.com/googlefonts/fontra/pull/1678/commits/ee0f88c15d8e58f49f22e8022f0daae59cb5e187)